src/EditorFeatures/Test/Collections/IntervalTreeTests.cs

```diff
@@ -6,36 +6,40 @@
 
 using System;
 using System.Collections.Generic;
+using System.Collections.Immutable;
 using System.Linq;
+using Microsoft.CodeAnalysis.Collections;
 using Microsoft.CodeAnalysis.Shared.Collections;
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.VisualStudio.Text;
+using Roslyn.Test.Utilities;
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.Editor.UnitTests.Collections;
 
-public sealed class IntervalTreeTests
+public abstract class IntervalTreeTests
```

Comment:
this allows me to run the same tests on both impls of interval trees.  note: this test code is not pretty.  I didn't want to spend a lot of time figuring out how to make it pretty.